pkg/cmd/gist/edit/edit.go

```diff
@@ -55,6 +56,7 @@ func NewCmdEdit(f *cmdutil.Factory, runF func(*EditOptions) error) *cobra.Comman
 		Args:  cmdutil.ExactArgs(1, "cannot edit: gist argument required"),
 		RunE: func(c *cobra.Command, args []string) error {
 			opts.Selector = args[0]
+			opts.IsPhysicalFile = false
```

Comment:
Is this field ever going to be true? If not, why would we need it?

Reply:
Not in gist viewing no. But in gist creation yes. I just want to standardize the bool values from the opts config itself if that's okay.